src/service/navigation.js

```diff
@@ -245,6 +253,35 @@ export class Navigation {
     this.handleNavClick_(e, target, location);
   }
 
+  /**
+   * Handles contextmenu click. Note that currently this only deals
+   * with url variable substitution and expansion, as there is
+   * straightforward way of determining what the user clicked in the
+   * context menu, required for A2A navigation and custom link protocol
+   * handling.
+   * TODO(alabiaga): investigate fix for handling A2A and custom link
+   * protocols.
+   * @param {!Element} target
+   */
+  handleContextmenuClick(target) {
```

Comment:
Remove and use `expandVarsForAnchor_` directly. You can put the comment in `handle_`.

Reply:
done